Src/Empty.ruleset

```diff
@@ -0,0 +1,601 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I specify this ruleset for all our tests (see [here](https://github.com/AutoFixture/AutoFixture/pull/773/commits/d75c05d479fe11bbb8ba4d1e3ffd857e0f1b270a#diff-948806e36ff188d70085b17290b13be1R7)). That is the only way I found to disable code analysis for the test projects.

Previously, we controlled that via the `RunCodeAnalysis>true</RunCodeAnalysis>` csproj property, but that no longer works .